packages/shared/tsup.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig((options) => ({
   },
   treeshake: true,
   sourcemap: true,
-  clean: true,
+  clean: false,
```

Comment:
Am I correct that this won't affect things in CI anyway if (assumption here) we have a clean environment we're starting with on each CI run?

Just wanted to make sure we won't have leftovers in the output directory that would affect future builds. Locally it seems like less of an option.

Reply:
@skitterm yes that's true when CI starts with a clean env. But even if it doesn't, I made sure `build` will do the cleaning up first:
https://github.com/chromaui/chromatic-e2e/pull/117/files#diff-3112d87d94a3b92a884111fabe12432fc0772999a6a024978f662e52919c28f3R43